### PR TITLE
GEOD-320 clean up site info component

### PR DESCRIPTION
**Pull request by heyagosper**

### CODE REVIEW

File: src/client/app/site-info/site-info.component.ts [hunk 1]
```diff
@@ -103,21 +83,11 @@ export class SiteInfoComponent implements OnInit, OnDestroy {
...
       gnssReceivers: [],
       surveyedLocalTies: [],
-      localEpisodicEvents: [],
-      humiditySensors: [],
-      pressureSensors: [],
-      temperatureSensors: [],
-      waterVaporSensors: []
     };
 
     this.siteLogOrigin = {
```
Comment (oehm-smith) [inline src/client/app/site-info/site-info.component.ts]: I don't think this is needed either, as long as it is declared.  Could you remove and try again.

File: src/client/app/site-info/site-info.component.ts [hunk 2]
```diff
@@ -180,11 +141,6 @@ export class SiteInfoComponent implements OnInit, OnDestroy {
           this.siteLogModel = {
```
Comment (oehm-smith) [inline src/client/app/site-info/site-info.component.ts]: I don't believe this is needed either, as long as it is declared.  Could you remove and test again.

File: src/client/app/site-info/site-info.component.ts (hunk 1 above)
Comment (heyagosper) [inline src/client/app/site-info/site-info.component.ts]: tests and UI appear to still work fine

**Comment by lbodor** [comment]
:+1:

**Pull request merged**